CoinDock-api/database/migrations/2022_05_18_103829_create_logs_table.php

```diff
@@ -34,8 +34,9 @@ public function up()
      */
     public function down()
     {
-        Schema::dropForeign('logs_user_id_foreign');
-        Schema::dropForeign('logs_wallet_id_foreign');
-        Schema::dropIfExists('logs');
```

Comment:
Is they any specific reason? to remove  Schema::dropIfExists('logs'); 

below code was handle only foreignkey relations. not table drop.

Reply:
I'm added like this

Schema::table('logs', function (Blueprint $table) {
            $table->dropForeign('logs_user_id_foreign');
            $table->dropForeign('logs_wallet_id_foreign');
        });